pkg/splunk/enterprise/indexercluster.go

```diff
@@ -154,6 +154,13 @@ func ApplyIndexerCluster(client splcommon.ControllerClient, cr *enterprisev1.Ind
 		cr.Status.IdxcPasswordChangedSecrets = make(map[string]bool)
 
 		result.Requeue = false
+		// Set indexer cluster CR as owner reference for clustermaster
+		namespacedName = types.NamespacedName{Namespace: cr.GetNamespace(), Name: GetSplunkStatefulsetName(SplunkClusterMaster, cr.Spec.ClusterMasterRef.Name)}
+		err = splctrl.SetStatefulSetOwnerRef(client, cr, namespacedName)
```

Comment:
NIT: Should we rename this to SetStatefulSetOwnerRef?

Reply:
isn't it already `SetStatefulSetOwnerRef`? :)

Reply:
@gaurav-splunk Sorry my bad. Corrected a typo. I meant **SetStatefulOwnerRef**. I only meant to avoid the word set twice.

Reply:
Will be taken up later